tern/analyze/common.py

```diff
@@ -164,6 +164,20 @@ def convert_to_pkg_dicts(pkg_dict):
     return pkg_list
 
 
+
+
+
+
+
+
+
+
+
+
+
+
+
+
```

Comment:
I think this was unintentional?

Reply:
Did that to check if prospector in tox caches it.